src/metabase/collections/api.clj

```diff
@@ -980,55 +981,11 @@
                   :can_restore
                   :can_delete)))
 
-(api.macros/defendpoint :get "/:id"
-  "Fetch a specific Collection with standard details added"
-  [{:keys [id]} :- [:map
-                    [:id ms/PositiveInt]]]
-  (collection-detail (api/read-check :model/Collection id)))
-
 (api.macros/defendpoint :get "/trash"
   "Fetch the trash collection, as in `/api/collection/:trash-id`"
   []
   (collection-detail (api/read-check (collection/trash-collection))))
 
-(api.macros/defendpoint :get "/:id/items"
```

Comment:
I need to move this to the bottom of the file, so `/collection/root/items` resolves before `/collection/:id/items`